docs/Configuration.md

```diff
@@ -2,107 +2,102 @@
-- **username** - the username to use in logging in to this server.
+- **username** - the username to use in logging in to this server
 - **password** - password for the specified username. Entering the password in this file is acceptable only in limited situations with very low need for security. 
 
-If you do not add a password to the server definition, anyone using the server needs to supply the password. Or, you can store the password securely in the system Keychain. The InterSystems Server Manager adds the following commands for managing stored passwords to the Command Palette:
+If you do not store the password securely in the system Keychain or add it to the server definition, anyone using the server needs to supply the password. The InterSystems Server Manager provides the following commands for managing stored passwords in the Command Palette:
 
-- **InterSystems Server Manager: Clear Password from Keychain** - remove password for selected server
-- **InterSystems Server Manager: Store Password in Keychain** - select a server and enter a password
-- **InterSystems Server Manager: Test Server Selection**
-- **InterSystems Server Manager: Test Server Selection (flush cached credentials)**
-- **InterSystems Server Manager: Test Server Selection with Details**
+- **InterSystems Server Manager: Clear Password from Keychain** - remove the password for a selected server
+- **InterSystems Server Manager: Store Password in Keychain** - select a server or create a new one and enter a password
 
 ## Configuring a Server Connection
 
-Select InterSystems ObjectScript from the settings editor extensions list. You need to edit the server configuration in the settings.json file, so your click *Edit in settings.json* under the heading **Objectscript: conn**. 
+Open the folder where you want client-side files to be located. Select the **ObjectScript Explorer** button which has been added to the Activity Bar. Select the **Choose Server and Namespace** button. This action opens a dialog that lets you select a server, or create a new one. Once you have selected a server and namespace, connection configuration is complete.
+
+## Editing a Server Connection
+
+If you need to modify a server connection select **File > Preferences > Settings** (**Code > Preferences > Settings** on Mac) from the menu. Select the **Workspace** settings level. Search for **objectscript: conn**, and click on *Edit in settings.json*.
 
-You should enter code something like this:
+The connection configuration looks like this:
 
-```js
+```json
 "objectscript.conn": {
 	"ns": "USER",
-	"server": "tst",
+	"server": "iris-1",
     "active": true,
 },
 ```
-The components of this server definition are:
+
+The components of this configuration are:
 
 - **ns** - namespace to use on the server
 - **server** - server name as specified in the server configuration
-- **active** - specifies whether the connection is active.
-
-## Configuring Export from Server
```

Comment:
Why this section is deleted?

Reply:
@daimor we are moving some content around to group it better, so that may be why, but @jstetson10 may be able to comment more.

Reply:
@daimor, in #409 I wrote re this section:

> The info under 'Configuring Export from Server' is really for advanced users in the situation where the default export behaviour doesn't meet their needs. Don't alarm newcomers with it.